pkg/demoinfocs/sendtables2/entity.go

```diff
@@ -477,14 +489,12 @@ func (p *Parser) OnPacketEntities(m *msgs2.CSVCMsg_PacketEntities) error {
 		index = next
 
 		cmd = r.readBits(2)
-		if cmd == 0 && m.GetHasPvsVisBits() > 0 {
+		if cmd == 0 && isPvsPacket {
 			cmd = r.readBits(2) << 3
-			if cmd&0x08 == 8 {
-				continue
-			}
 		}
-		if cmd&0x01 == 0 {
-			if cmd&0x02 != 0 {
+
+		if cmd&updateFlagLeave == 0 {
```

Comment:
to reviewdog: it was already "complex" 😭

I can "flatten" it or extract handlers to other functions, that was on my mind first, but I thought people in go don't like too many functions.

Reply:
I will take a look when I'm on my PC 🙂

Personally I like having lots of functions in Go (I know this project doesn't look like it as a lot of it is REd and kinda done 1:1) but let's leave it for now until I had a chance to take a proper look 😄

Reply:
PS: reviewdog is more of a guideline than hard rules, so we don't always have to listen to it 😄

Reply:
Oh, I already made this way - inspired by game code.
![ezgif com-video-to-gif_2](https://github.com/markus-wa/demoinfocs-golang/assets/9150636/70e1cb20-892e-4f01-a81b-18ad6dce560b)